widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -164,6 +166,14 @@ class ChatView(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defSty
                 super.onScrolled(recyclerView, dx, dy)
                 controller?.onRecyclerviewPositionChanged(!recyclerView.canScrollVertically(1))
             }
+
+            override fun onScrollStateChanged(recyclerView: RecyclerView, newState: Int) {
+                super.onScrollStateChanged(recyclerView, newState)
+
+                // hide the keyboard on chat scroll
+                context.getSystemService<InputMethodManager>()
```

Comment:
Please make this into a extension function or a util method. This will probably be needed elsewhere as well

Reply:
we have extensions already defined.
you can use `insetsController?.hideKeyboard()` which is used in this class multiple times.

Reply:
fixed!